pylabnet/launchers/launch_control.py

```diff
@@ -714,24 +714,47 @@ def _update_debug_level(self, i=0):
         # pylabnet_server, pylabnet_gui, launcher
         self.debug_level = self.main_window.debug_comboBox.currentText()
 
-    def _start_stop_logging(self):
-        """ Starts or stops logging to file depending on situation """
+    def start_stop_logging(self, master_log=False):
+        """ Starts or stops logging to file depending on situation
 
-        if self.main_window.logfile_status_button.isChecked():
+        :master_log: (bool) If True, this function is called as initial setup function of
+            filesaving for the master launch control. In this case a log path as specified
+            in the config file is chosen.
+        """
+
+        if self.main_window.logfile_status_button.isChecked() or master_log:
 
             # Actually start logging
             filename = f'logfile_{datetime.now().strftime("%H_%M_%S")}'
-            try:
+
+            # Get logging file from json.
+            filepath = None
+            if master_log:
+                try:
+                    config_dict = load_config('static_proxy')
+                    filepath = config_dict['logger_path']
+                except:
+                    self.main_window.terminal.setText('Critical error: '
+                                                      'no logger_path found in static_proxy.json')
```

Comment:
Added a debug statement - this kind of thing is important for when we want to test stuff on other machines where we will inevitably forget to make the config file and/or make an incorrect config file. This way we can rapidly identify the mistake that is causing the launcher to crash.